README.md

```diff
@@ -139,3 +139,19 @@ On Xenial
 On Bionic
 
     rocker --nvidia --x11 osrf/ros:melodic-desktop-full gazebo
+
+## Volume mount
+
+`--volume` adds paths as docker volumes. The last path must be terminated with two dashes `--`.
+
+    rocker --volume ~/.vimrc ~/.bashrc -- ubuntu:18.04
+
+The above example of the volume option will be expanded in `docker run` as
+
+    --volume ~/.vimrc:~/.vimrc --volume ~/.bashrc:~/.bashrc  
```

Comment:
Thanks for the iteration, but the other important part of this expansion is that it will resolve `~/` to `/home/<USERNAME>/` via abspath. Docker will abort if you try to pass non-absolute paths.